Kernel/Syscalls/mmap.cpp

```diff
@@ -489,9 +490,40 @@ int Process::sys$munmap(void* addr, size_t size)
         return 0;
     }
 
-    // FIXME: We should also support munmap() across multiple regions. (#175)
+    // Try again while checkin multiple regions at a time
+    // slow: without caching
+    const auto& regions = space().find_regions_intersecting(range_to_unmap);
+    // if there still no regions found error out
+    if (!regions.size())
+        return -EINVAL;
```

Comment:
This decides *against* what POSIX says. I think this is perfectly reasonable, and we should do this in Serenity, so: Yeah, let's do this.

Reply:
This is a reamnant from the original implementation, but ok...